test/e2e/framework/helper/validation/certificaterequests/certificaterequests.go

```diff
@@ -39,10 +46,15 @@ func ExpectDuration(duration, fuzz time.Duration) func(certificaterequest *cmapi
 			return err
 		}
 
+		// Here we ensure that the requested duration is what is signed on the
+		// certificate. We tolerate fuzz either way.
 		certDuration := cert.NotAfter.Sub(cert.NotBefore)
-		if certDuration > (duration+fuzz) || certDuration < duration {
-			return fmt.Errorf("expected duration of %s, got %s (fuzz: %s) [NotBefore: %s, NotAfter: %s]", duration, certDuration,
-				fuzz, cert.NotBefore.Format(time.RFC3339), cert.NotAfter.Format(time.RFC3339))
+		if certDuration > (expectedDuration+fuzz) || certDuration < (expectedDuration-fuzz) {
```

Comment:
I made the fuzz allowance symmetrical (similar to [certificatesigningrequests.go](https://github.com/cert-manager/cert-manager/pull/7774/files#diff-5e85f8e0340be3d78f73772f504bd299550acc81c9c9642e8e2c858432a35e33)).